### ISSUE TITLE
Publicly exposed Vuplex 3D WebView

### ISSUE THREAD

**Issue by binarynate** [Khizrkshah/VrShopThesisProject#1]
Hello 👋🏻. This is Nate, the Founder of [Vuplex](https://vuplex.com). First, thank you for using 3D WebView! I really appreciate your business. I'm opening this issue because your repo includes the assets for [3D WebView](https://vuplex.com), which aren't allowed to be shared publicly according to [its license](https://support.vuplex.com/articles/commercial-library-license). To resolve this issue, can you please make the repository private?

https://github.com/Khizrkshah/VrShopThesisProject/tree/a8124bb5bd2885f7c947bb62f9846695b39bdc4f/Assets/Vuplex/WebView

**Comment by Khizrkshah** [comment]
Yes sure i can make it private, till i add the vuplex asset to the git ignore :)
